frontend/src/components/PostDetailComponent.vue

```diff
@@ -45,28 +49,38 @@ export default {
             alt="Post Image"
           />
         </div>
-        <div class="mt-8 mr-0 mb-1 ml-8 flex m-0">
-          <p class="font-bold">Category:&nbsp</p>
-          <router-link
-            :to="{
-              name: 'categoryPosts',
-              params: { slug: postData.category.slug },
-            }"
-            class="text-black no-underline"
-          >
-            {{ postData.category.name }}
-          </router-link>
-        </div>
-        <div class="ml-8 flex m-0">
-          <p class="font-bold">Tags:&nbsp</p>
-          <router-link
-            v-for="tag in postData.tags"
-            :to="{ name: 'posts', query: { tag: tag.slug } }"
-            :key="tag.slug"
-            class="text-black no-underline"
-          >
-            {{ tag.name }},&nbsp;
-          </router-link>
+        <div class="mt-4 w-full">
+          <div class="w-full">
+            <div class="mr-0 mb-1 ml-8 flex m-0">
+              <p class="font-bold">Category:&nbsp</p>
```

Comment:
ich würde diese `&nbsp` entfernen und auf dem p-tag die class `mr-1` verwenden. 
Hat noch andere Stellen in deinem Code, kommt 3x vor